src/subcommands/utils.go

```diff
@@ -0,0 +1,54 @@
+package subcommands
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	pb "accretional.com/semantifly/proto/accretional.com/semantifly/proto"
+	"google.golang.org/protobuf/proto"
+)
+
+func readIndex(indexFilePath string, index *pb.Index, ignoreIfNotFound ...bool) error {
```

Comment:
What is the purpose of ignoreIfNotFound and why do we pass it in like this when we only ever check the first element?

Reply:
`ignoreIfNotFound` is when the `index.list` file is not found during the read. That flag controls whether to throw out an error (for the case of `update` or `get` commands) or to ignore it (for the case of `add` where we will create a new index file).

I was trying out how to pass an optional parameter in Go but this was the only way suggested. I will remove the variadic args and just make it a boolean arg